MobiusExtras/Test/MobiusViewControllerTests.swift

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2019 Spotify AB.
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#if canImport(UIKit)
+
+import Foundation
+import MobiusCore
+import MobiusExtras
+import Nimble
+import Quick
+
+final class MobiusViewControllerTests: QuickSpec {
+    override func spec() {
+        context("View Controller using Mobius") {
```

Comment:
These tests don't actually test that calls to `viewDidAppear` and `viewDidDisappear` are forwarded. I could not think of a way to do this.

Reply:
That seems proper, testing that UIKit works as documented isn’t a unit test. :-)